src/httpClient/httpClient.ts

```diff
@@ -348,6 +355,10 @@ export interface HttpClientOptions {
    * @link https://github.com/JustinBeckwith/retry-axios/blob/v2.6.0/src/index.ts#L11
    */
   retryOptions?: RetryConfig;
+    /**
+   * Timeout option
```

Comment:
Can you link the axios documentation for the timeout option, so that it is obvious what values can be set and what they mean? I.e. 0 is no timeout, right?

Reply:
done